errai-demos/errai-bus-demo-stress-test/pom.xml

```diff
@@ -333,13 +334,14 @@
         <dependency><groupId>io.netty</groupId><artifactId>netty-codec-http</artifactId><scope>provided</scope></dependency>
         <dependency><groupId>io.netty</groupId><artifactId>netty-handler</artifactId><scope>provided</scope></dependency>
 
-        <!-- And finally, add this -->
+        &lt;!&ndash; And finally, add this &ndash;&gt;
         <dependency>
           <groupId>org.jboss.errai</groupId>
           <artifactId>errai-jboss-as-support</artifactId>
           <scope>compile</scope>
         </dependency>
       </dependencies>
+-->
     </profile>
```

Comment:
Since this is a comment block now, we should probably remove the escaped `&lt;!&ndash;` characters too.

Reply:
@treblereel as Alex is on PTO, could you please fix this?

Reply:
@edewit sure
@tiagobento yeap, currently i am working to fix it in separate PR https://github.com/errai/errai/pull/400

Reply:
@treblereel you mean @ederign instead of me ;)